desktop/src/app.rs

```diff
@@ -99,6 +101,8 @@ impl App {
             loaded = LoadingState::Loaded;
         }
 
+        let mut gilrs = Gilrs::new().ok();
```

Comment:
Anything worth logging here?
In particular, if user explicitly asks for gamepad settings but we don't manage to initialize?